src/bgg_companion_game_filter.py

```diff
@@ -0,0 +1,22 @@
+from src.models.BoardGame import BoardGame
+from src.models.GameFilter import GameFilter
+
+
+class FilterBoardGame:
+
+    def game_matches_filter(self, game: BoardGame, gamefilter: GameFilter) -> bool:
```

Comment:
nitpick: I suppose this could also be a static method, since none of the functionality depends on the state of `FilterBoardGame`, This class seems like properly houses all of the logic needed to filter games, and can just be a wrapper around each of the filters for the class.

Reply:
Hahah I was thinking through this actually, just pushed up something.  Originally I wrote this by actually instantiating the FilterBoardGame class WITH game and gamefilter, but than that didn't really seem to serve a purpose.  Because I was still needing to pass these things into the function I was classing within the class.

Reply:
Yeah that's another way you could do it! Then `game_matches_filter` would call the requisite filters with `self.game` and `self.gamefilter`.

Reply:
Ahhhhhh okay I see yeah I was accidentally mixing both together.  But if I wanted to instantiate the class with game and game filter,   `game_matches_filter` doesn't need to have parameters passed in because it's getting it from that instance of FilterBoardGame class

Reply:
This is a nice refresher for me on classes.  One of the things I don't feel too confident with

Reply:
And I guess in that case, game_matches_filter would not be a static method because it cares about the state of the class

Reply:
Exactly! Or, again, you can make `game_matches_filter` static and take the parameters. Either way would totally work!